chart/dapr-ambient/templates/helmhooks.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ include "dapr-ambient.fullname" . }}-hook
+  labels:
+    {{- include "dapr-ambient.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": pre-delete
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  # This shows how to use a simple value. This will look for a passed-in value
+  # called restartPolicy. If it is not found, it will use the default value.
+  # {{default "Never" .restartPolicy}} is a slightly optimized version of the
+  # more conventional syntax: {{.restartPolicy | default "Never"}}
+  restartPolicy: "Never"
+  serviceAccountName: {{ include "dapr-ambient.serviceAccountName" . }}
+  containers:
+  - name: ambient-remove-hook
+    image: "{{ .Values.ambient.initContainer.image.registry }}/{{ .Values.ambient.initContainer.image.name }}:{{ .Values.ambient.initContainer.image.tag }}"
+    args: 
+      - remove
+      - --config-map={{ .Values.ambient.configMapName }}
```

Comment:
@mcruzdev there is no need to have `ambient.configMapName` as a parameter to the chart. Instead of that we can use a convention, something like `{{ .Release.name }}-cm` That will use the helm release name + -cm to create the configMap name. You will need to replace this also in the other places where this is referenced.